doc-gen/package.json

```diff
@@ -0,0 +1,73 @@
+{
+    "name": "ak-docgen",
+    "version": "3.18.2",
+    "description": "A CLI tool used to generate .mdx documentation for jsdoc public methods",
+    "author": "Alchemy",
+    "license": "MIT",
+    "private": true,
+    "type": "module",
+    "main": "./dist/esm/exports/index.js",
+    "types": "./dist/types/exports/index.d.ts",
+    "typings": "./dist/types/exports/index.d.ts",
+    "sideEffects": false,
+    "files": [
+      "dist",
+      "src/**/*.ts",
+      "!dist/**/*.tsbuildinfo",
+      "!vitest.config.ts",
+      "!.env",
+      "!src/**/*.test.ts",
+      "!src/__tests__/**/*"
+    ],
+    "exports": {
+      ".": {
+        "types": "./dist/types/exports/index.d.ts",
+        "default": "./dist/esm/exports/index.js"
+      },
+      "./package.json": "./package.json"
+    },
+    "bin": {
+      "ak-docgen": "./dist/esm/cli.js"
+    },
+    "scripts": {
+      "prebuild": "node --loader ts-node/esm ./inject-version.ts",
+      "build": "yarn clean && yarn build:esm && yarn build:types",
+      "build:esm": "tsc --project tsconfig.build.json --module es2015 --outDir ./dist/esm --removeComments && echo > ./dist/esm/package.json '{\"type\":\"module\"}'",
```

Comment:
If we do go through with the permissions PR (https://github.com/alchemyplatform/aa-sdk/pull/718), we'll probably want to change the module to es2020 to be consistent.

Reply:
ah yea good call, lemme approve yours and then I'll update that here